pkg/aggregator/aggregator.go

```diff
@@ -257,3 +267,72 @@ func mergeSpecs(dest, source *spec.Swagger, renameModelConflicts, ignorePathConf
 	}
 	return nil
 }
+
+// deepEqualDefinitionsModuloGVKs compares a and b, but ignores the x-kubernetes-group-version-kind extension.
+// It will mutate a temporarily, but not b.
+func deepEqualDefinitionsModuloGVKs(a, b *spec.Schema) bool {
+	old, oldFound := a.ExtraProps["x-kubernetes-group-version-kind"]
```

Comment:
ExtraProps is not serialized and not what we use to store GVK. it is spec.Schema.VendorExtensible

Reply:
good point. Fixed.